storm-core/src/jvm/org/apache/storm/Config.java

```diff
@@ -140,6 +140,12 @@
     public static final String STORM_META_SERIALIZATION_DELEGATE = "storm.meta.serialization.delegate";
 
     /**
+     * A list of daemon statistics  reporter plugin class names.
+     */
+    @isStringList
+    public static final String STORM_STATISTICS_PREPARABLE_REPORTER_PLUGINS = "storm.statistics.preparable.reporter.plugins";
```

Comment:
Reposting since github ate my comment:

Not sure myself, but how about `storm.daemon.metrics.reporter.plugins` ?

Wanted to specify that this was for daemon metrics and make it plural.